go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/cosmos/cosmos-proto v1.0.0-alpha7
 	github.com/cosmos/cosmos-sdk/db v1.0.0-beta.1
 	github.com/cosmos/go-bip39 v1.0.0
-	github.com/cosmos/iavl v0.18.0
+	github.com/cosmos/iavl v0.17.3
```

Comment:
We have to downgrade right? Isn't IAVL 0.18 what contains all the performance improvements @marbar3778?

Reply:
could use the osmosis v 0.17 ?

I think that the mainline 18 is osmo 17 with a few other changes iirc

Reply:
it does not. 0.18 has 0.35 tendermint which we need to remove

Reply:
also I flubbed, the osmo fork, would require further changes and cause possible further sadness.